addons/ofxiOS/src/sound/SoundInputStream.m

```diff
@@ -116,43 +116,19 @@ - (void)start {
 	AVAudioSession * audioSession = [AVAudioSession sharedInstance];
 	NSError * err = nil;
 	
-    #ifdef __IPHONE_6_0
 	// need to configure set the audio category, and override to it route the audio to the speaker
 	if([audioSession respondsToSelector:@selector(setCategory:withOptions:error:)]) {
 		// we're on iOS 6 or greater, so use the AVFoundation API
 		if(![audioSession setCategory:AVAudioSessionCategoryPlayAndRecord
-						  withOptions:AVAudioSessionCategoryOptionDefaultToSpeaker
+						  withOptions:AVAudioSessionCategoryOptionMixWithOthers
 								error:&err]) {
```

Comment:
@danoli3 - curious about this change.  Does that mean for all iOS apps going forward they want capture exclusive audio out? ( I know this is in the input stream - but I remember there being some issues related to this way back )